SC_redesign/Crop_and_Soil/crop/dormancy.py

```diff
@@ -0,0 +1,77 @@
+from typing import Optional
+
+from SC_redesign.Crop_and_Soil.crop.crop_data import CropData, PlantTypes
+
+"""
+This module is based on the "Dormancy" module of SWAT (5:1.2)
+"""
+
+
+class Dormancy:
+    def __init__(self, crop_data: Optional[CropData] = None):
+        self.data = crop_data or CropData
+
+    def go_into_dormancy(self) -> None:
+        """This method performs the actual transition from active to dormant in a crop.
```

Comment:
```suggestion
        """
        Performs the actual transition from active to dormant in a crop.
```
Does it mean that there is a 'non-actual' transition as well?

Reply:
Yeah, kind of. In `crop.py`, there is the `start_dormancy()` function that calls this function, and `field.py` contains the function `assess_dormancy()` that calls `start_dormancy()` on every crop in a field. So this makes it exceedingly clear where the transition from the non-dormant to the dormant state is happening. I can change the wording if you think it doesn't read well though.